### ISSUE TITLE
License compatibility

### ISSUE THREAD

**Issue by mosteo** [alire-project/alr#7]
Side project that would enable search for compatible licensed projects. Current search already can be used on license abbreviations.

**Comment by onox** [comment]
Additionally a new command `check licenses` could check license compatibility using the spdx crate (already in use by Alire). Similar to what the 'deny' plugin of cargo does. See https://github.com/EmbarkStudios/cargo-deny#licenses.

**Comment by mosteo** [comment]
Yes, this is in the spirit of the idea. Also `alr with` could warn/fail when you try to add an incompatible-licensed dependency to the ones already in use.
